docs/source/design/authoring.rst

```diff
@@ -65,26 +65,26 @@ There is only one :py:class:`LaunchPlan <flytekit.core.launch_plan.LaunchPlan>`
 .. autoclass:: flytekit.core.launch_plan.LaunchPlan
    :noindex:
 
+.. exception_handling:
+
 ******************
 Exception Handling
 ******************
-Exception handling is done along two dimensions:
+Exception handling takes place along two dimensions:
 
-* System vs User: We try to differentiate between user exceptions and Flytekit/system level exceptions. For instance, if Flytekit
-  fails to upload its outputs, that's a system exception. If you the user raise a ``ValueError`` because of unexpected input
-  in the task code, that's a user exception.
-* Recoverable vs Non-recoverable: Recoverable errors are retried and counted against your task's retry count. Non-recoverable errors just fail. System exceptions, by default, are recoverable (since there's a good chance it was just a blip).
+* System vs User: We try to differentiate between user exceptions and Flytekit/system level exceptions. For instance, if Flytekit fails to upload its outputs, that's a system exception. If the user raises a ``ValueError`` because of unexpected input in the task code, that's a user exception.
+* Recoverable vs Non-recoverable: Recoverable errors will be retried and count against your task's retry count. Non-recoverable errors will simply fail. System exceptions are by default recoverable (since there's a good chance it was just a blip).
```

Comment:
```suggestion
* System vs. User: We try to differentiate between user exceptions and Flytekit/system-level exceptions. For instance, if Flytekit fails to upload its outputs, that's a system exception. If the user raises a ``ValueError`` because of an unexpected input in the task code, that's a user exception.
* Recoverable vs. Non-recoverable: Recoverable errors will be retried and counted against the task's retry count. Non-recoverable errors will simply fail. System exceptions are by default recoverable (since there's a good chance it was just a blip).
```